modules/010-deckhouse-flow-schema/templates/flowschema.yaml

```diff
@@ -0,0 +1,191 @@
+# Schema generated by "tools/flow_schema.go" DO NOT EDIT.
+# To generate run 'make generate'
+---
+apiVersion: flowcontrol.apiserver.k8s.io/v1beta2
+kind: FlowSchema
+metadata:
+  name: deckhouse-flow-schemas
+  {{- include "helm_lib_module_labels" (list . (dict "app" .Chart.Name)) | nindent 2 }}
+spec:
+  distinguisherMethod:
+    type: ByUser
+  matchingPrecedence: 1000
+  priorityLevelConfiguration:
+    name: deckhouse-prioritylevel
+  rules:
+  - resourceRules:
+    - apiGroups: [""]
+      clusterScope: true
```

Comment:
What does this option mean? Will requests to list secrets in a namespace not be limited?

Reply:
`clusterScope` indicates whether to match requests that do not
specify a namespace (which happens either because the resource
is not namespaced or the request targets all namespaces).
If this field is omitted or false then the `namespaces` field
must contain a non-empty list.

`namespaces` is a list of target namespaces that restricts
matches.  A request that specifies a target namespace matches
only if either (a) this list contains that target namespace or
(b) this list contains "*".  Note that "*" matches any
specified namespace but does not match a request that _does
not specify_ a namespace (see the `clusterScope` field for that).

So to select namespaced and non-namespaced resources we should use 
clusterScope: true
namespaces: *